src/server/graphql/models/User/userMutation.js

```diff
@@ -151,7 +151,13 @@ export default {
         createdAt: new Date(userInfo.created_at),
         tms: userInfo.tms
       };
-      const {email, id: userId, picture, preferredName} = auth0User;
+      const {email, id: userId, createdAt, picture, preferredName} = auth0User;
```

Comment:
Hmm. Ok. I feel like I had originally taken that route but was met by doom and gloom by the time the event stream got to our tool. But I will refactor and we can test.